pkgs/top-level/php-packages.nix

```diff
@@ -53,10 +53,10 @@ lib.makeScope pkgs.newScope (self: with self; {
     inherit (pkgs) stdenv autoreconfHook fetchurl re2c;
   };
 
-  # Wrap mkDerivation to prepend pname with "php-" to make names consistent
+  # Prepend name with "php-" to make names consistent
   # with how buildPecl does it and make the file easier to overview.
-  mkDerivation = { pname, ... }@args: pkgs.stdenv.mkDerivation (args // {
-    pname = "php-${pname}";
+  mkDerivation = attrs: (pkgs.stdenv.mkDerivation attrs).overrideAttrs(previousAttrs: {
+    name = "php-${previousAttrs.pname}-${previousAttrs.version}";
```

Comment:
I do not really like this much – `name` is terrible concept and IMO should not exist other than as a label for store path for easier debugging. As such, I would like to hide it from Nixpkgs as much as possible. But yeah, it is not really clear who should be responsible for it, then.

But I guess if we set it here, it should probably refer to `finalAttrs` – if someone updates a version with `overrideAttrs`, the store path name should reflect that.